src/bicep/modules/hubNetwork.bicep

```diff
@@ -240,6 +240,7 @@ module azureMonitorPrivateLink './privateLink.bicep' = if ( contains(supportedCl
     logAnalyticsWorkspaceResourceId: logAnalyticsWorkspaceResourceId
     privateEndpointSubnetName: subnetName
     privateEndpointVnetName: virtualNetwork.outputs.name
+    location: location
```

Comment:
This PR contains some changes related to how location is set on policy and networking. They do not seem related to the purpose of the PR. Did you intend to include these changes.? They appear to make sense, but maybe they are part of another effort?

Reply:
Good catch! Yes, these changes are intentional.

These changes are the cost of updating to Bicep v0.4.1272 which introduces new warnings for implied resource locations.

> Makes updates to source to fix explicit-values-for-loc-params Bicep warnings that occur when upgrading Bicep to v0.4.1272

You can see what happens without these changes in this failed `validate-build-bicep` run here:

https://github.com/Azure/missionlz/runs/5208013963?check_suite_focus=true#step:3:19

Reply:
👍 that makes sense. It looks great. 😻 I'm running some test deployments.